docs/roles-policies/policies/certification/determining-case-outcome.adoc

```diff
@@ -8,83 +8,100 @@
-The second one also determines if a case that was rejected (or not approved) at stage N continues in stage N+1 or not.
+== Introduction
 
+Each case is reviewed in one or more stages, by one or more reviewers.
 
-== Stage-level outcome determination strategies
+When determining the outcome of a certification campaign, midPoint has to consider several factors:
 
-Currently, there are four strategies defined:
+* How to evaluate varying responses from individual reviewers.
+* How to evaluate cases in which no reviewers are assigned (e.g. if a reviewer is defined as the target role owner, but a particular role has no owner), or when reviewers fail to provide any answers during a stage.
+* Which of the following cases should be considered.
+    ** Those that were explicitly accepted during a previous phase.
+    ** Those that were not rejected.
+    ** All cases.
+* How should results from individual stages be combined.
+
+There are two parameters that address the factors above:
+
+* Stage-level-defined `*outcomeStrategy*` - Determines how decisions of reviewers within a stage are combined into "stage decision".
+* Campaign-level-defined `*reviewStrategy*` - Controls how decisions taken at individual stages combine into the final decision.
+It also determines if a case that was rejected (or not approved) at stage N continues in stage N+1 or not.
+
+== Outcome determination strategies
+
+=== Stage level
+
+The following strategies control how outcome is determined:
 
 [%autowidth]
 |===
 | Strategy | Description
 
-| oneAcceptAccepts
+| <<OneAcceptAccepts,`oneAcceptAccepts`>>
 | If at least one reviewer accepts, the result is "Accept" regardless of the other votes.
 This is the default strategy.
 
 
-| allMustAccept
+| <<AllMustAccept,`allMustAccept`>>
 | All reviewers must accept.
-No reviewers mean No response.
+No reviewers means No response.
 
 
-| oneDenyDenies
+| <<OneDenyDenies,`oneDenyDenies`>>
 | If at least one reviewer denies (either via revoke or reduce), the result is not approved.
-However, for the case to be approved, at least one approval must be present.
+However, for a case to be approved, at least one approval must be present.
 
 
-| acceptedIfNotDenied
+| <<AcceptedIfNotDenied,`acceptedIfNotDenied`>>
 | Approved if none of the reviewers denies (either via revoke or reduce).
-So e.g. if nobody says anything, the case is approved.
+For example, if nobody acts, the case is approved.
 
 
 |===
 
-Moreover, one can define a default response that is used when no reviewers are available.
-(This doesn't mean that no reviewers responded! It means that e.g. reviewers were defined as target role owners, but a particular role has no owners.)
+You can define the default response that is used when no reviewers are available.
+This does not apply to cases when reviewers do not respond, but rather to situations such as when reviewers are defined as target role owners but a particular role has no owners.
```

Comment:
Original text has definition of the behavior and an example. You are shortening it to an example only :-/